bin/build

```diff
@@ -14,7 +14,7 @@ BRANCH="${TRAVIS_BRANCH:-`git rev-parse --abbrev-ref HEAD | tr / -`}"
 echo Branch: $BRANCH
 
 # ...same for the commit.
-COMMIT="${TRAVIS_COMMIT:-`git rev-parse HEAD`}"
+COMMIT="${TRAVIS_COMMIT:-`git rev-parse --short HEAD`}"
```

Comment:
does the ${TRAVIS_COMMIT:....} with the --short flag work for you? it didn't for me with gaia. i had to use COMMIT=`git rev-parse --short HEAD`

Reply:
I'll let you know as soon as develop builds.